sawmill-core/pom.xml

```diff
@@ -10,7 +10,7 @@
     <parent>
         <artifactId>sawmill</artifactId>
         <groupId>io.logz.sawmill</groupId>
-        <version>1.1.42</version>
+        <version>1.1.43</version>
```

Comment:
@MarinaRazumovsky @oririechman I've noticed that a popular pattern is being used for versioning libraries:
1. They use a 0-SNAPSHOT version in all pom.xml files
2. In travis they define a deploy stage that is activated only when a new GitHub release is created. It runs a a script which changes the maven version to the GitHub release name (e.g. 1.2.4) and then uploads it to the repository. 
See: 
* https://github.com/testcontainers/testcontainers-java/blob/master/.travis.yml
* https://maven.apache.org/maven-ci-friendly.html

Thus we can also utilise the Release tab of GitHub for keeping track of release notes.